### PR TITLE
RDM-6323: Functional Tests for 'Validate a set of fields as Case worker' operation

### PR DESCRIPTION
**Pull request by SivaKanukolluQA**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-6323

### Change description ###
Implement Functional Tests for 'Validate a set of fields as Case worker' operation

**Does this PR introduce a breaking change?** (check one with "x")
[ ] Yes
[x] No

### CODE REVIEW

File: src/aat/resources/features/F-050/F-050.feature [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    Then a negative response is received
+    And the response [has the 403 return code]
+    And the response has all other details as expected
+
+  @S-299
+  Scenario: must return negative response when request does not provide valid authentication credentials
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [uses an invalid user Id that doesn’t exist in CCD]
+    And the request [uses an invalid authorization]
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-050/F-050.feature]: [uses a dummy Authorization token]

File: src/aat/resources/features/F-050/S-301.td.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          "AddressLine1": null,
+          "AddressLine2": null,
+          "AddressLine3": null,
+          "PostTown": null,
+          "County": null,
+          "PostCode": null,
+          "Country": null
+        }
+      }
+    }
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-050/S-301.td.json]: This data object seems to be reusable from a common file

Comment (SivaKanukolluQA) [reply src/aat/resources/features/F-050/S-301.td.json]: Created a new data file for common data object.

**Comment by MSancaktutar** [comment]
Merged myself assessing the code good enough and observing the green build.

**Pull request merged**